### PR TITLE
Make audio transfer lossless

### PR DESCRIPTION
**Pull request by chappjo**
This is my attempt to make the audio transfer lossless. It works for me, but it definitely needs more testing as I am not (yet) experienced with python. Please merge with caution.

### CODE REVIEW

File: inference_video.py [hunk 1]
```diff
@@ -14,22 +14,19 @@
 def transferAudio(sourceVideo, targetVideo):    
     import shutil
     import moviepy.editor
-    tempAudioFileName = "./temp/audio.mp3"
+    tempAudioFileName = "./temp/audio.mkv"
```
Comment (chappjo) [inline inference_video.py:17]: Swith to mkv from mp3 as mkv can handle more codecs. If however a codec that mkv doesn't support it used, the transferAudio function may fail.

**Comment by hzwer** [comment]
@SleepingFox88 What do you think of this modification？

**Comment by chappjo** [comment]
Okay, I think I've found an issue, sometimes the audio might not be supported in the output container. I these case a fallback should exist. I'm trying to add that now. Sorry about all this, as I said, I'm new to python.

**Comment by chappjo** [comment]
That should fix the issue I found.

Before if you were to convert a file with WMA audio into an mp4 file it would fail, as WMA is not supported in the mp4 container. Now whenever the video and audio are not merged, it will fallback to mp3.

**Comment by hzwer** [comment]
👍

**Comment by SleepingFox88** [comment]
> @SleepingFox88 What do you think of this modification？

Looks good to me.

**Pull request merged**